adapters/repos/db/helpers/helpers.go

```diff
@@ -71,3 +71,7 @@ func BucketSearchableFromPropNameLSM(propName string) string {
 func BucketRangeableFromPropNameLSM(propName string) string {
 	return BucketFromPropNameLSM(propName + "_rangeable")
 }
+
+func IsVectorBucket(bucketName string) bool {
```

Comment:
should `XYZ_muvera_vectors` be included?

Reply:
yeah ther was a discussion if we gonna include or not, but seems we shall include it